qos-monitor/src/main/java/eu/arrowhead/core/qos/dto/event/monitoringevents/ReceivedMonitoringRequestEventDTO.java

```diff
@@ -17,4 +21,14 @@ public ReceivedMonitoringRequestEventDTO() {
 		this.eventType = QosMonitorEventType.RECEIVED_MONITORING_REQUEST;
 	}
 
+	//-------------------------------------------------------------------------------------------------
+	@Override
+	public String toString() {
+		return new StringJoiner(", ", InterruptedMonitoringMeasurementEventDTO.class.getSimpleName() + "[", "]")
```

Comment:
InterruptedMonitoringMeasurementEventDTO.class.getSimpleName() is wrong here

Reply:
done